backends/ebpf/targets/kernel_target.py

```diff
@@ -54,8 +54,10 @@ def compile_dataplane(self):
             args += f" INCLUDES+=-include{self.options.extern} "
             # need to include the temporary dir because of the tmp import
             args += f" INCLUDES+=-I{self.tmpdir} "
-        errmsg = "Failed to compile the eBPF byte code:"
-        return testutils.exec_process(args, errmsg).returncode
+        result = testutils.exec_process(args)
+        if result.returncode != testutils.SUCCESS:
```

Comment:
this repeated code suggests that you want a function that takes a message and a command and logs the message if the command failed

Reply:
I weighed off both of these options and I felt that keeping the `exec_process` class simple is more beneficial. Most of these error messages are redundant.